readme.md

```diff
@@ -210,6 +313,9 @@ Once installation is working, take a look at the existing code to make sure you
 
 * **Your Answer:**
 
+TODO
+
```

Comment:
- Status 200 received with response which identifies the user associated to the token. We are authenticated/authorized by way of the token submitted along with the HTTP request.